### PR TITLE
Completed assignment

### PR DESCRIPTION
**Pull request by jonathanstelman**
The second, pruned tree has a 24% error rate, compared with the 33% error rate of the first model. This is because the first model is overfit, and produces much greater error when presented with a test set vs. a training set.

### CODE REVIEW

**Comment by charles-lang** [comment]
Looks good. Nice work!
